bootstrap/14-networking.sh

```diff
@@ -50,3 +50,13 @@ fi
 if ${dns_is_set}; then
     chroot_exec resolvconf -u
 fi
+
+if [ ! -f ${ETC}/hostname ]; then
```

Comment:
We don't have to check if the file exists. It can cause the problem when the user specified the hostname but, for some reason, `/etc/hostname` exists and the changes wouldn't be reflected.